pages/emailSender.tsx

```diff
@@ -165,95 +188,110 @@ const EmailSender: NextPage = () => {
             <StyledSubHeader variant="h5">
-              2) Upload and import csv
+              3) Verify final messages
             </StyledSubHeader>
-            <StyledCsvButtonsContainer>
-              <input
-                style={{ display: 'none' }}
-                id="contained-button-file"
-                accept={'.csv'}
-                type="file"
-                onChange={handleOnChange}
-              />
-              <label htmlFor="contained-button-file">
-                <Button variant="contained" component="span">
-                  Upload
-                </Button>
-              </label>
-              <StyledCsvButton
-                variant="contained"
-                width="medium"
-                onClick={(e) => {
-                  handleOnSubmit(e)
-                }}
-              >
-                Import CSV!
-              </StyledCsvButton>
-            </StyledCsvButtonsContainer>
-          </SectionContainer>
-        </FormControl>
-        <StyledTableContainer>
-          <TableContainer component={Paper}>
-            <StyledTable aria-label="simple table">
-              <TableHead>
-                {headerKeys.map((key) => (
-                  <TableCell key={nanoid()}>
-                    <StyledBoldTypograhy variant="body1">
-                      {key}
-                    </StyledBoldTypograhy>
-                  </TableCell>
-                ))}
-              </TableHead>
-              <TableBody>
-                {csvRowsArray.map((item) => (
-                  <StyledTableRow key={nanoid()}>
-                    {Object.values(item).map((val) => (
-                      <TableCell key={nanoid()} align="left">
-                        {val}
-                      </TableCell>
-                    ))}
-                  </StyledTableRow>
-                ))}
-              </TableBody>
-            </StyledTable>
-          </TableContainer>
-        </StyledTableContainer>
-        <SectionContainer>
-          <StyledSubHeader variant="h5">
-            3) Verify final messages
-          </StyledSubHeader>
-          <StyledButton
-            color="info"
-            variant="contained"
-            onClick={createMessages}
-            width="medium"
-          >
-            Print final messages
-          </StyledButton>
-          <StyledFinalMessagesContainer>
-            {displayMessages()}
-          </StyledFinalMessagesContainer>
-        </SectionContainer>
-      </StyledPageContainer>
-    </ThemeProvider>
+            <StyledButton
+              color="info"
+              variant="contained"
+              onClick={createMessages}
+              width="medium"
+            >
+              Print final messages
+            </StyledButton>
+            </SectionContainer>
+            <br />
+            <br />
+            <SectionContainer>
+            <StyledSubHeader variant="h5">4) Send emails</StyledSubHeader>
+            <StyledButton
```

Comment:
Same as above- let's hide/disable the buttons on the page until they are actionable